packages/contracts/contracts/EpochKeyLiteProofVerifier.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IVerifier} from './interfaces/IVerifier.sol';
+import {BaseVerifier} from './BaseVerifier.sol';
+
+contract EpochKeyLiteProofVerifier is BaseVerifier {
+    constructor(IVerifier _verifier) BaseVerifier(_verifier) {}
```

Comment:
You should double check, but I think you shouldn't need to explicitly define this constructor. Try just removing this line and seeing if it works as expected.

Reply:
The contract doesn't compile if the constructor isn't explicitly defined